version.py

```diff
@@ -26,4 +26,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = '49.1'
+PROVISION_VERSION = '51.0'
```

Comment:
Why not 50.0?

Reply:
There are two commits, each needing a major bump.